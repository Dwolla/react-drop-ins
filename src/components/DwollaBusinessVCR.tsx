import type { FC } from 'react';
import type { BusinessVCRProps } from '../types/component-props';

const DwollaBusinessVCR: FC<BusinessVCRProps> = (props) => {
  return <dwolla-business-vcr {...props} />;
};

export default DwollaBusinessVCR;
