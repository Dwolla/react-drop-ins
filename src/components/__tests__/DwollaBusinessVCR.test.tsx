import TestRenderer from 'react-test-renderer';
import DwollaBusinessVCR from '../DwollaBusinessVCR';

test('DwollaBusinessVCR', () => {
  const tree = TestRenderer.create(<DwollaBusinessVCR />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('with optional props', () => {
  const tree = TestRenderer.create(
    <DwollaBusinessVCR
      correlationId="123abc"
      email="janedoe@email.com"
      firstName="Jane"
      lastName="Doe"
      ipAddress="255.255.255.255"
      hideDBAField
      privacy="www.yourprivacy.com"
      terms="www.yourterms.com"
      customerId="123"
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
