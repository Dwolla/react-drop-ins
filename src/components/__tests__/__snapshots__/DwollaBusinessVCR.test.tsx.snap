// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaBusinessVCR 1`] = `<dwolla-business-vcr />`;

exports[`with optional props 1`] = `
<dwolla-business-vcr
  correlationId="123abc"
  customerId="123"
  email="janedoe@email.com"
  firstName="Jane"
  hideDBAField={true}
  ipAddress="255.255.255.255"
  lastName="Doe"
  privacy="www.yourprivacy.com"
  terms="www.yourterms.com"
/>
`;
